pallets/schemas/src/runtime-api/Cargo.toml

```diff
@@ -10,21 +10,17 @@ repository = "https://github.com/libertyDSNP/frequency/"
 edition = "2021"
 
 [dependencies]
-codec = { package = "parity-scale-codec", version = "3.6.1", default-features = false, features = [
```

Comment:
Rename to parity-scale-codec, allows removing `codec` alias from root `Cargo.toml`